webapi/Services/SemanticKernelProvider.cs

```diff
@@ -159,7 +162,9 @@ IMemoryStore CreateMemoryStore()
             switch (memoryOptions.Retrieval.VectorDbType)
             {
                 case string x when x.Equals("SimpleVectorDb", StringComparison.OrdinalIgnoreCase):
-                    return new VolatileMemoryStore();
+                    // Maintain single instance of volitile memory.
```

Comment:
Typo in volatile

```suggestion
                    // Maintain single instance of volatile memory.
```

Reply:
Of course...always.  One day I'll learn 👍